docs/OncoKB-Data-Access.md

```diff
@@ -1,8 +1,8 @@
 # Introduction
 
-A data access token is required to use OncoKB. Please review OncoKB [terms](https://www.oncokb.org/terms) 
-and request for [data access](https://www.oncokb.org/dataAccess) if you would like to use OncoKB data 
-with your private cBioPortal instance.
+A data access token is required for versions `v3.2.4` and greater to use OncoKB. Please review 
```

Comment:
@onursumer should be `v3.2.3` right?

Reply:
never mind - looks like i forgot to commit the upgrade to kubernetes repo. So looks like backend was reverted to v3.2.2 somehow.

Reply:
seems like `v3.2.3` is released already